cartridge-cli.lua

```diff
@@ -2999,6 +3000,14 @@ function cmd_pack.parse(arg)
     args.type = parameters[1]
     args.path = parameters[2]
 
+    if args.path == nil then
+        args.path = '.'
```

Comment:
Why not `fio.cwd()`?

Reply:
Wow, thanks a lot, I missed this function! 
I will fix it anywhere in the code.